Build/Build.cs

```diff
@@ -259,6 +259,7 @@ void ReportTestOutcome(params string[] globFilters)
                     ReportTypes.lcov,
                     ReportTypes.HtmlInline_AzurePipelines_Dark)
                 .AddFileFilters("-*.g.cs")
+                .AddFileFilters("-**/.nuget/**")
```

Comment:
Maybe I am wrong, but I think this should be: `-*.nuget*.cs`, since it supports wildcard but not glob filters.

Reply:
I think you're right.

Reply:
Hmm.. apparently I wasn't.. I don't how to handle this then, because this behind the scenes is converted to a regex and and a `Regex.IsMatch()` is performed:

https://github.com/danielpalme/ReportGenerator/blob/15c6fc1004070c26689d4c30110b81d05d256544/src/ReportGenerator.Core/Parser/Filtering/DefaultFilter.cs#L73-L99

Reply:
I think it's fixed now.